chef/automate
# Reverted the changes done for the NFS mount Service

### :nut_and_bolt: Description: What code changed, and why?
- Did changes on NFS Mount service such that it could check for on-prem Self-managed as well as its not supported so gave the response like that 
- Also refactored the httpsutils as it written twice refactored the complete code piece


### :chains: Related Resources
- https://chefio.atlassian.net/browse/CHEF-3819
### :+1: Definition of Done

### :athletic_shoe: How to Build and Test the Change
- Checkout this branch
- Rebuild components/automate-cli
- Make an setup for on-prem setup 
- Run `chef-automate verify --config config.toml`
### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
- The Response from the trigger as the its not able to unmarshal the response coming from the service

<img width="1792" alt="Screenshot 2023-06-30 at 9 11 13 PM" src="https://github.com/chef/automate/assets/62262938/4a88683a-aee8-485f-8b9c-90512d041144">



- The response form the Handler of the NFS mount Service Handler as AWS and Self-managed is not supported by the NFS Mount 


<img width="1792" alt="Screenshot 2023-06-30 at 8 31 54 PM" src="https://github.com/chef/automate/assets/62262938/c99cc33f-87ec-47b4-ab2a-1edfec5c3a3f">
